### DISCUSSION TITLE
Problemas com decimal usando o Console.ReadLine();

### DISCUSSION THREAD

**Discussion by rveigar** [balta-io/2801#51] [Perguntas e Respostas] [answered]
Eu fiz a calculadora conforme o video do Balta. 
        Console.WriteLine("Digite o primeiro valor: ");
        float v1 = float.Parse(Console.ReadLine());
       Console.WriteLine("Digite o segundo valor: ");
        var v2 = float.Parse(Console.ReadLine()); 

sempre que eu digito um número usando o ponto, por exemplo 3.22 e mando imprimir a variável, é mostrado um inteiro, no caso, 322.
Então eu pensei que se digitasse com vírgula em vez de ponto funcionaria, e realmente funciona. Será que isso ocorre porque meu Windows está em português, com as configurações regionais do Brasil? Eu acho que sim.
Então eu resolvi colocar uma variável float no meu programa apenas para verificar se é ponto ou vírgula.
      var v3 = 1.22f;
      Console.WriteLine("Valor de v3: "+v3);

e o valor mostrado de v3 é 1,22 e não 1.22.  (mostra com vírgula e não com ponto)

se eu tento colocar v3 usando vírgula, o VSCode não deixa.
     var v3 = 1,22f

Sendo assim, fiquei sem entender nada. Na digitação do valor via Console.ReadLine eu tenho que usar virgula, se eu colocar um valor hardcoded, eu tenho  que utilizar ponto. 

Não sei se fui claro. Alguém poderia me explicar o que está acontecendo?

saída do programa:

== Multiplicacao
Digite o primeiro valor: 
1,22
Valor de v3: 1,22       
VAlor de v1: 1,22       
Digite o segundo valor: 
1,22
VAlor de v2: 1,22
O valor da multiplicação de 1,22 por 1,22 é 1,4884001

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @rveigar, como vai?
Já temos um discussão para este assunto aqui:
https://github.com/balta-io/2801/discussions/36
